libcaf_core/caf/typed_event_based_actor.test.cpp

```diff
@@ -287,14 +286,17 @@ TEST("chainging the behavior at runtime and skipping messages") {
 }
 
 TEST("starting a string delegator chain") {
-  auto aut = sys.spawn(string_delegator, sys.spawn(string_reverter), true);
+  auto reverter = sys.spawn(string_reverter);
+  auto delegator = sys.spawn(string_delegator, reverter);
+  auto aut = sys.spawn(string_delegator, delegator);
   SECTION("message_types() returns an interface description") {
     std::set<std::string> iface{"(std::string) -> (std::string)"};
     check_eq(aut->message_types(), iface);
   }
-  // inject().with("Hello World!"s).from(self).to(aut);
-  // dispatch_message();
-  // expect<std::string>().with("!dlroW olleH"s).to(self);
+  inject().with("Hello World!"s).from(self).to(aut);
+  expect<std::string>().with(std::ignore).from(self).to(delegator);
+  expect<std::string>().with(std::ignore).from(self).to(reverter);
```

Comment:
Don't add `with` if you don't care anyways.

```suggestion
  expect<std::string>().from(self).to(delegator);
  expect<std::string>().from(self).to(reverter);
```